packages/zoe/src/contractSupport/topics.js

```diff
@@ -7,7 +7,7 @@ export { SubscriberShape };
 export const PublicTopicShape = M.splitRecord(
   {
     subscriber: SubscriberShape,
-    storagePath: M.promise(/* string */),
+    storagePath: M.or(M.promise(/* string */), M.string()),
```

Comment:
As we discussed, changing this shape is safe against upgrade as it couldn't have been used as a durable state of collection shape before because durably storing promises is not possible.